API.md

```diff
@@ -356,7 +356,7 @@ When using triggerEvent to upload a file the `eventType` must be `change` and yo
 triggerEvent(
   'input.fileUpload',
   'change',
-  [new Blob(['Ember Rules!'])]
+  { files: [new Blob(['Ember Rules!'])] }
```

Comment:
Oh, I just realized this changes the `API.md` directly. That file is autogenerated and shouldn't be manually modified.

Its built from the inline API docs, which for this example I think means here:

https://github.com/emberjs/ember-test-helpers/blob/f1927fb4fb396d7e96db4596f650f8aba523cb31/addon-test-support/%40ember/test-helpers/dom/trigger-event.ts#L22-L25